jekyll/_cci2/circleci-images.md

```diff
@@ -25,8 +25,10 @@ This document provides information about pre-built CircleCI images and a listing
 {:.no_toc}
 
 For convenience, CircleCI maintains several Docker images. These images are
-typically extensions of official Docker images and include tools especially
-useful for CI/CD. This document will provide an overview of best practices when using a convenience image. Please note that we advise using the **next-generation** images convenience images rather than **legacy images** (as explained below).
+typically extensions of official Docker images, and include tools especially
+useful for CI/CD.
+
+This document provides an overview of best practices when using a convenience image. Please note that we advise using the **next-generation** images (these start `cimg/`) convenience images rather than **legacy images**, as explained below.
```

Comment:
```suggestion
This document provides an overview of best practices when using a convenience image. Please note that we advise using the **next-generation** convenience images (these start `cimg/`) rather than **legacy images**, as explained below.
```